src/sparseml/modifiers/utils/pytorch_helpers.py

```diff
@@ -85,3 +93,41 @@ def run_calibration_forward(
         batch = tensors_to_device(batch, model_device)
         with torch.no_grad():
             forward_fn(batch, module=model)
+        check_for_undercalibrated_modules(
```

Comment:
let's only run this after calibration

Reply:
the other place we can put it is right before we call `freeze` - this will also cover QAT